packages/strapi-design-system/src/TextButton/TextButton.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import styled, { keyframes } from 'styled-components';
+import Loader from '@strapi/icons/Loader';
 import { Box } from '../Box';
 import { Typography } from '../Typography';
 import { Flex } from '../Flex';
 import { buttonFocusStyle } from '../themes/utils';
 
+const rotation = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(359deg);
+  }
+`;
+
+const LoadingWrapper = styled.div`
+  animation: ${rotation} 2s infinite linear;
```

Comment:
Could you please add a `will-change: transform;` here too, to animate it smoothly? If that is missing from the others could you add it there too? :)

Reply:
interesting I didn't know about this one
from the documentation I see it should be used only when really necessary/last resort on performance issues, do you think we really need it here? 

<img width="796" alt="image" src="https://user-images.githubusercontent.com/71838159/188593968-aff820bb-8908-4a4c-8116-74200ba73aa6.png">

Reply:
In this case it will do a good job, because it will move the element into it's own compositor and therefore make sure it always renders at 60fps.

Reply:
okay good to know, on it!